x-pack/plugins/ingest_pipelines/public/application/sections/pipelines_list/details_flyout.tsx

```diff
@@ -186,6 +187,23 @@ export const PipelineDetailsFlyout: FunctionComponent<Props> = ({
               </EuiDescriptionListDescription>
             </>
           )}
+
+          {/* Metadata (optional) */}
+          {pipeline._meta && (
+            <>
+              <EuiDescriptionListTitle data-test-subj="metaTitle">
+                <FormattedMessage
+                  id="xpack.ingestPipelines.list.pipelineDetails.metaDescriptionListTitle"
+                  defaultMessage="Metadata"
+                />
+              </EuiDescriptionListTitle>
+              <EuiDescriptionListDescription>
+                <EuiCodeBlock language="json">
+                  {JSON.stringify(pipeline._meta, null, 2)}
```

Comment:
I think here we could potentially also use the same stringify function defined at `x-pack/plugins/ingest_pipelines/public/application/lib/utils.ts ` after is refactored, right?

Reply:
Done.